ruby/example_code/rds/list_cluster_snapshots.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[rds.ruby.listClusterSnapshots]
+require "aws-sdk-rds"  # v2: require 'aws-sdk'
+
+# List all snapshots for an Amazon Relational Database Service (Amazon RDS)
+# Multi-AZ DB cluster.
```

Comment:
List all snapshots for all Amazon Relational Database Service (Amazon RDS) clusters.

Reply:
Wait...so you think this call would be inclusive of Aurora clusters? Now I have to check...